downloads/smaht-submitr-install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+BRANCH=master
+BRANCH=pyinstaller-experiment-20240611
+BASEURL=https://raw.githubusercontent.com/smaht-dac/submitr/$BRANCH/downloads
```

Comment:
I see why you did it this way, but I would prefer if you used the release mechanism rather than putting it directory in the source code tree ie: https://github.com/smaht-dac/submitr/releases